src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 40;
+
```

Comment:
Можно добавить переменную и не делать рассчёты по нескольку раз)
```suggestion
const count = days * cost;
```